.github/workflows/release.yml

```diff
@@ -43,3 +43,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NPM_CONFIG_PROVENANCE: ${{ secrets.NPM_CONFIG_PROVENANCE }}
```

Comment:
why an secret for this?

Reply:
```
permissions:
      contents: read
      id-token: write
```

is missing for oidc connection

Reply:
Oh yeah, we could just set it there directly.

> permissions:
      contents: read
      id-token: write
      
> is missing for oidc connection

Could you commit the appropiate changes? I think that will be faster as you're have more knowledge in that area.

Reply:
Reading the [npm documentation](https://docs.npmjs.com/generating-provenance-statements#using-third-party-package-publishing-tools) this should be enough.